.urlignore

```diff
@@ -9,4 +9,4 @@ https://polkadot.com/
 # Ignore Google Fonts service
 https://fonts.gstatic.com/
 https://fonts.googleapis.com/
-https://portal.grove.city
+https://portal.grove.city/
```

Comment:
Maybe add a comment above this like # All other links or something or a line break at the very least so it's clear it's separate from the google fonts links

```suggestion

https://portal.grove.city/
```